docs/docs/tutorial-basics/boundaries.md

```diff
@@ -6,7 +6,23 @@ sidebar_position: 3
 
 > Key concepts: Clamping, animations (handlers)
 
+## Resources
+
+- [Clamp](https://docs.swmansion.com/react-native-reanimated/docs/utilities/clamp/)
+
+## Challenge
+
+→ We need to keep him inside the screen boundaries.
+
+→ We want to drop him and make him bounce on the ground.
+
 <video controls autoplay loop style={{maxHeight: 640, minHeight: 200}}>
 
   <source src="https://user-images.githubusercontent.com/80724668/187310576-f6cf041b-3428-4989-94b2-c34346e66945.mov" />
 </video>
+
+💡 TIPS && HINTS:
```

Comment:
Docusaurus has a cool feature here where we can create blocks that collapse to "hide" the hints until the reader wants them.


```
<details>
<summary>💡 TIPS && HINTS:</summary>

- You already have the gesture handler you need for this, consider using callbacks.
- The library provides us a [clamp](https://docs.swmansion.com/react-native-reanimated/docs/utilities/clamp) function to bound the movement.
- You can explore the `withTiming` config for the bouncing.

</details>
```

![image](https://github.com/carozo/slimy/assets/3135138/4fa46fa0-ceac-4718-8bec-1075b53479ab)
![image](https://github.com/carozo/slimy/assets/3135138/9a7ef528-6541-4c48-869b-504791dd66f5)